src/content/about-carbon/principles.md

```diff
@@ -0,0 +1,28 @@
+---
+title: About Carbon
+label: Principles
+---
+
+# Principles
+
+These core principles drive everything we design at IBM. By relentlessly focusing on our users and their needs, we can ensure that our products and experiences deliver useful, meaningful, and delightful outcomes.  
+
```

Comment:
Can remove `# Principles` since the h1 is generated by the code above. Also, do you want to set the page intro styles on that first paragph? 

`<page-intro>These core principles drive everything we design at IBM. By relentlessly focusing on our users and their needs, we can ensure that our products and experiences deliver useful, meaningful, and delightful outcomes.  </page-intro>

That just adds the bold styling we have across the pages for our intro paragraphs
<img width="753" alt="screen shot 2018-09-19 at 3 57 20 pm" src="https://user-images.githubusercontent.com/2753488/45781091-b79a7e00-bc24-11e8-88f1-16dc07447fd7.png">